### DISCUSSION TITLE
How to actually use the "accent" color from the official theme designer?

### DISCUSSION THREAD

**Discussion by tcdw** [tamagui/tamagui#2837] [General]
Well, I tried to use the [theme designer](https://tamagui.dev/theme), and it seems there would be a variant called "accent" for different kind of UI components. However, I have no idea with how to actually use the "accent" color in my Expo project.

Or, how do I make the "continue" button as that "accent" color in the screenshots by myself, with the configuration I made on the right side?

![image](https://github.com/user-attachments/assets/ca8f199e-da09-4a75-aef5-c281caf46dff)

**Comment by marcos-credfranco** [comment]
Hello, how are you? You can use [Theme](https://tamagui.dev/docs/core/theme), with the name 'accent', or, in the case of the button, use the theme attribute (theme='accent') of the button itself.
